[fix]: [PL-32635]: Adds note for vault v2 engine characters limit for secret name (#1636)
harness/developer-hub: docs/platform/Secrets/2-add-use-text-secrets.md

MySecret:<+secrets.getValue("test")>  
EOF
```

### Review: Characters length limit in Secret Names

> If you are using Vault V2 engine, you can not use secret name with more than 100 characters.

Reviewer: ```suggestion
A secret name cannot exceed 100 characters if you are using the Vault V2 engine.
```

Author: Done
